sdk/rust/nym-sdk/examples/geo_topology_provider.rs

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: Apache-2.0
+
+use nym_client_core::client::topology_control::geo_aware_provider::{
```

Comment:
We'd probably want to re-export those types as part of the sdk

Reply:
Yes good idea